diagrams/request_uri_mode_post.md

```diff
@@ -0,0 +1,38 @@
+```plantuml
+@startuml
+
+autonumber
+
+participant "Wallet" as w
+
+participant "User Agent" as u
+
+participant "Verifier" as r
+
+u --> r : use
+activate r
+
+r --> u: authorization request\n(client_id, request_uri, request_uri_method=post, [client_id_scheme])
+deactivate r
+u --> w: authorization request\n(client_id, request_uri, request_uri_method=post, [client_id_scheme])
+activate w
```

Comment:
```suggestion
activate w
w --> w check client_id with trust framework

Note that the client_id is self asserted by the verifier. If it is true and trusted then no harm will come to the user. However, if the client_id is not trusted, whether it is true or not, then the user should be informed that an untrusted verifier is requesting information and asked if he/she wants to proceed. Finally if the client_id is false but identifies a trusted verifier, then it is vital that the request_uri that is responded to is the one that actually belongs to the trusted client_id, and is provided by the trust framework, and is not the request_uri provided in the request

```

Reply:
I meant that the client_id is falsely claimed by the verifier to belong to itself, when it does not. It belongs to a trusted verifier.